content_scripts/link_hints.coffee

```diff
@@ -620,9 +620,9 @@ LocalHints =
         isClickable ||=
           if element == document.body and document.hasFocus() and Scroller.isScrollableElement element
             reason = "Scroll."
-      when "div", "ol", "ul"
+      when "div", "ol", "ul", "pre"
         isClickable ||=
-          if element.clientHeight < element.scrollHeight and Scroller.isScrollableElement element
+          if (element.clientHeight < element.scrollHeight or element.clientWidth < element.scrollWidth) and Scroller.isScrollableElement element
```

Comment:
I tested with current prod version v.s. my local and it works...https://jsfiddle.net/w9vasqf5/1/
But yeah you are right the default direction is y, plz give me some time to find it out. BTW, I didn't see similar test cases for this, could you help advise on it? thanks!